src/components/Modals/ReplaceCourseModal.vue

```diff
@@ -0,0 +1,294 @@
+<template>
+  <teleport-modal
+    title="Replace Course"
+    content-class="content-course"
+    :leftButtonText="leftButtonText"
+    :rightButtonText="rightButtonText"
+    :rightButtonIsDisabled="selectedCourse == null"
+    @modal-closed="closeCurrentModal"
+    @left-button-clicked="backOrCancel"
+    @right-button-clicked="addItem"
+  >
+    <div class="newCourse-text">
+      {{ selecting ? 'Search Course Roster' : 'Selected Course' }}
```

Comment:
Can this be a computed value? Same with the other expressions and magic strings in this file